lib/JSONtoXD.ts

```diff
@@ -1,6 +1,35 @@
 import type { Clue, CrosswordJSON } from "./types"
 import { replaceWordWithSymbol } from "./xdparser2"
 
+/**
+ *
+ * Takes CrosswordJSON's meta.rebus property, a Clue, and a split character to properly
+ * rebuild a clue answer.
+ *
+ * The clue is needed for the answer and the split locations (in case there are splits)
+ *
+ * The meta.rebus is needed because sometimes the split locations aren't mapped correctly if the rebus causes
+ * the clue answer to be of a different length
+ *
+ * inserts the rebus symbol where it needs to be, adds the splits in the proper locations, then replaces the
+ * rebus symbol with the word
+ *
+ * @param {Object} getXDClueAnswerProps - has CrosswordJSON, clue and splitChar
+ * @property {CrosswordJSON} getXDClueAnswerObject.json - CrosswordJSON information that includes meta.rebus
+ * @property {Clue} getXDClueAnswerObject.clue - Clue that includes split locations
+ * @property {string} getXDClueAnswerObject.splitChar - split character to insert
+ * @returns {string} clue answer in it's xd format form
+ */
+export function getXDClueAnswer({ json, clue, splitChar }: { json: CrosswordJSON, clue: Clue, splitChar: string }) {
```

Comment:
Can you make this function act like the rest of this repo please - e.g. inline arguments to functions